timescaledb/overview/core-concepts/scaling.md

```diff
@@ -104,10 +113,9 @@ access node, such that aggregates are performed on data nodes
 whenever possible, and only aggregated or filtered results are passed
 back to the access node for merging and returning to a client.
 
-
 [data model]: /overview/data-model-flexibility/
 [chunking]: https://www.timescale.com/blog/time-series-data-why-and-how-to-use-a-relational-database-instead-of-nosql-d0cd6975e87c
-[Compression Operational Overview]: /how-to-guides/compression/
+[compression operational overview]: /how-to-guides/compression/
```

Comment:
```suggestion
```

I searched for the link text and it doesn't look like this is actually linked from anywhere, let's just scrap it